DESCRIPTION

```diff
@@ -41,7 +41,8 @@ Imports:
     ggrepel,
     RColorBrewer,
     heatmaply,
-    gganimate
+    gganimate,
+    plotly
```

Comment:
Are we still using `shiny` after this PR? (I know you haven't touched heatmap code yet).

Reply:
We don't use shiny anymore. I'll delete codes regrading shiny.